components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint.util/src/main/java/org/wso2/carbon/identity/application/authentication/endpoint/util/AuthenticationEndpointUtil.java

```diff
@@ -435,4 +444,26 @@ private static void setAuthorizationHeader(HttpRequestBase httpMethod) {
 
         httpMethod.addHeader(HTTPConstants.HEADER_AUTHORIZATION, CLIENT + authHeader);
     }
+
+    /**
+     * Resolve "${tenantDomain}" in the URL.
+     *
+     * @param   url URL to be tenant resolved
```

Comment:
Shall we add a full stop after the each line of the comment to follow the same as other comments. Check the other places as well